rest/route/project_test.go

```diff
@@ -1240,3 +1241,222 @@ func (s *ProjectPutRotateSuite) TestRotateProjectVars() {
+			body := []byte(
+				`{
+                     "task_name": "t1",
+                     "build_variant": "bv1",
+                     "start_time": "2022-11-02T00:00:00.000Z",
+                     "end_time": "2022-11-03T00:00:00.000Z"
+                 }`)
+			req, _ := http.NewRequest(http.MethodGet, "https://example.com/api/rest/v2/projects/myProject/executions", bytes.NewBuffer(body))
+			req = gimlet.SetURLVars(req, map[string]string{"project_id": "myProject"})
+
+			err := rm.Parse(context.Background(), req)
+			assert.NoError(t, err)
+		},
+		"parseInvalidRequester": func(t *testing.T, rm *getProjectTaskExecutionsHandler) {
+			body := []byte(
+				`{
+                     "task_name": "t1",
+                     "build_variant": "bv1",
+                     "requesters": ["what_am_i"],
+                     "start_time": "2022-11-02T00:00:00.000Z",
+                     "end_time": "2022-11-03T00:00:00.000Z"
+                 }`)
+			req, _ := http.NewRequest(http.MethodGet, "https://example.com/api/rest/v2/projects/proj/num_excutions", bytes.NewBuffer(body))
+			req = gimlet.SetURLVars(req, map[string]string{"project_id": "proj"})
+
+			err := rm.Parse(context.Background(), req)
+			assert.Error(t, err)
+		},
+		"parseInvalidTime": func(t *testing.T, rm *getProjectTaskExecutionsHandler) {
+			body := []byte(
+				`{
+                     "task_name": "t1",
+                     "build_variant": "bv1",
+                     "requesters": ["gitter_request"],
+                     "start_time": "2022-11-02T00:00:00",
+                     "end_time": "2022-11-03T00:00:00"
+                 }`)
+			req, _ := http.NewRequest(http.MethodGet, "https://example.com/api/rest/v2/projects/myProject/excutions", bytes.NewBuffer(body))
+			req = gimlet.SetURLVars(req, map[string]string{"project_id": "myProject"})
+
+			err := rm.Parse(context.Background(), req)
+			assert.Error(t, err)
+		},
+		"parseInvalidTimeRange": func(t *testing.T, rm *getProjectTaskExecutionsHandler) {
+			body := []byte(
+				`{
+                     "task_name": "t1",
+                     "build_variant": "bv1",
+                     "requesters": ["gitter_request"],
+                     "start_time": "2022-11-04T00:00:00.000Z",
+                     "end_time": "2022-11-03T00:00:00.000Z"
+                 }`)
+			req, _ := http.NewRequest(http.MethodGet, "https://example.com/api/rest/v2/projects/myProject/excutions", bytes.NewBuffer(body))
+			req = gimlet.SetURLVars(req, map[string]string{"project_id": "myProject"})
+
+			err := rm.Parse(context.Background(), req)
+			assert.Error(t, err)
+		},
+		"parseNoBuildVariant": func(t *testing.T, rm *getProjectTaskExecutionsHandler) {
+			body := []byte(
+				`{
+                     "task_name": "t1",
+                     "requesters": ["gitter_request"],
+                     "start_time": "2022-11-02T00:00:00.000Z",
+                     "end_time": "2022-11-03T00:00:00.000Z"
```

Comment:
Can we please add two test cases, one with no end time and one with no start time so that we can make sure that parse handles them as expected?